tts/global.def.lua

```diff
@@ -42,3 +42,34 @@ function printToAll(message, color) end
 ---@return string
 function stringColorToRGB(player_color) end
 
+
+---@shape tts__SnapPoint
+---@field position tts__Vector Position of the snap point. The position is relative to the entity's center (a local position).
+---@field rotation tts__Vector Rotation of the snap point. The rotation is relative to the entity's rotation (a local rotation).
+---@field rotation_snap boolean If the snap point is a "rotation" snap point.
```

Comment:
I know it wasn't the case, and that you've just copied and paste from another file. However for consistency, could you please start the comments (post-type) with `@`.

Reply:
Fixed.